src/test/java/com/jcabi/github/mock/MkIssueEventsTest.java

```diff
@@ -0,0 +1,170 @@
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void createsIssueEventWithLabel() throws Exception {
+        final MkIssueEvents events = this.issueEvents();
+        final String label = "my label";
+        final Event event = events.create("labeled", 2, "samuel", label);
+        MatcherAssert.assertThat(
+            // @checkstyle MultipleStringLiteralsCheck (1 line)
+            event.json().getJsonObject("label").getString("name"),
+            Matchers.equalTo(label)
+        );
+    }
+
+    /**
+     * MkIssueEvents can get a single issue event.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void getsIssueEvent() throws Exception {
+        final MkIssueEvents events = this.issueEvents();
+        final String type = "unlocked";
+        final String login = "jill";
+        final int eventnum = events.create(type, 2, login, null).number();
+        final Event.Smart event = new Event.Smart(events.get(eventnum));
+        MatcherAssert.assertThat(
+            event.number(),
+            Matchers.equalTo(eventnum)
+        );
+        MatcherAssert.assertThat(
+            event.type(),
+            Matchers.equalTo(type)
+        );
+        MatcherAssert.assertThat(
+            event.author().login(),
+            Matchers.equalTo(login)
+        );
+    }
+
+    /**
+     * MkIssueEvents can list issue events in correct order.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void iteratesIssueEvents() throws Exception {
+        final MkIssueEvents events = this.issueEvents();
+        final Event first = events.create("closed", 3, "john", null);
+        final Event second = events.create("reopened", 3, "jane", null);
+        MatcherAssert.assertThat(
+            events.iterate(),
+            Matchers.<Event>iterableWithSize(2)
+        );
+        final Iterator<Event> iter = events.iterate().iterator();
+        MatcherAssert.assertThat(
+            iter.next(),
+            Matchers.equalTo(first)
+        );
+        MatcherAssert.assertThat(
+            iter.next(),
+            Matchers.equalTo(second)
+        );
+    }
+
+    /**
+     * Create an MkIssueEvents to work with.
+     * Can't use normal IssueEvents because we need the mock-only
+     * {@link MkIssueEvents#create(String, int, String, String)} method.
+     * @return MkIssueEvents
+     * @throws Exception If some problem inside
+     */
+    private MkIssueEvents issueEvents() throws Exception {
+        return (MkIssueEvents) (new MkGithub().repos().create(
+            Json.createObjectBuilder().add("name", "test").build()
+        ).issueEvents());
+    }
+
+    /**
+     * Obtains the current time.
+     * @return Current time (in milliseconds since epoch) truncated to the nearest second
+     */
+    private static long now() {
+        final long sinceepoch = new Date().getTime();
+        return sinceepoch - (sinceepoch % Tv.THOUSAND);
```

Comment:
@cvrebert got it. let's use `System.currentTimeMillis()` instead of `new Date().getTime()`.